business/MetadataDatabase/Data/UserAuthenticatedDto.cs

```diff
@@ -0,0 +1,12 @@
+namespace MetadataDatabase.Data
+{
+    public class UserAuthenticatedDto
```

Comment:
For use inheritance for UserAuthenticatedDto and UserRegisterDto
` public class UserAuthenticatedDto : UserDto
    {
        public string token { get; set; }
    }`

Reply:
You're right